test/abort/test-http-parser-consume.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const http = require('http');
+const spawn = require('child_process').spawn;
+
+if (process.argv[2] === 'child') {
+  const server = http.createServer(common.mustCall((req, res) => {
+    res.end('hello');
+  }));
+
+  server.listen(0, common.mustCall((s) => {
+    const rr = http.get({ port: server.address().port },
```

Comment:
I know people have asked for a 1e6 changes... but, would you reindent with both arguments in a new line (nececitating less whitespace):
```js
  server.listen(0, common.mustCall((s) => {
    const rr = http.get(
      { port: server.address().port },
      common.mustCall((d) => {
        rr.parser.consume(0);
        server.close();
      }));
    })
  );
```
I'm fine if you don't want to.